src/docs/theme.mdx

```diff
@@ -68,4 +68,31 @@ const Test = () => {
 }
 ```
 
+## Add font assets
+
+You can provide a second parameter to the `useTheme` hook to add font assets
+
+```js
+import React from "react";
+import { defaultLightTheme, useTheme } from "pi-ui";
+
+import TomorrowLight from "../assets/fonts/tomorrow/Tomorrow-Light.ttf";
+import TomorrowRegular from "../assets/fonts/tomorrow/Tomorrow-Regular.ttf";
+import TomorrowSemiBold from "../assets/fonts/tomorrow/Tomorrow-SemiBold.ttf";
+
+const fontConfig = {
+  fontFamilyText: "Tomorrow",
+  regularUrl: TomorrowRegular,
+  semiBoldUrl: TomorrowSemiBold,
+  lightUrl: TomorrowLight,
+  format: "truetype" // .ttf assets => 'truetype'
+};
+
+const App = () => {
+  defaultLightTheme["font-family-text"] = "Tomorrow";
```

Comment:
We could use `fontConfig.fontFamilyText` in `useTheme` instead of expecting the consumer to override the `font-family-text` property manually, what do you think :thinking: ?

Reply:
I guess we shouldn't, if we do use the `fontConfig`, consumer will need to provide it with each `useTheme` call which is not great, so maybe overriding the property on the default theme/ custom theme is the way to go.

Reply:
I think we need to set the `fontConfig.fontFamilyText` as the default in useTheme (e.g without the need of the consumer explicitly doing that). Otherwise, it won't be applied to the components, right?

Reply:
As discussed & decided we should create a standalone hooks just for the font config with the following usage:
`const [fontCfg, setFontCfg] = useFont();`